gradio/utils.py

```diff
@@ -352,3 +354,49 @@ def format_ner_list(input_string: str, ner_groups: Dict[str : str | int]):
 
     output.append((input_string[end:], None))
     return output
+
+
+class Http:
+    """
+    A class that shares an AsyncClient which sends and handles network requests.
+    Further different functionalities can be gathered within this class.
+    - Client configurations
+    - Response validation
+    - Support error types and error handling
+    - Client stop/restart
+    """
+
+    client = AsyncClient()
+
+    @classmethod
+    async def request(
+        cls,
+        method: str,
```

Comment:
We can use aiohttp's hdrs.METH_ALL type instead of str. 
`
from aiohttp import  hdrs
 ....
async def request(
        cls,
        method: hdrs.METH_ALL, ... ):
...
`

Reply:
Thx for the suggestion!

I actually thought about using `hdrs` however since we only need "get" and "post" I thought there is no need to add aiohttp to the requirements. Planning to only have `httpx` withing requirements.